src/utils/index.ts

```diff
@@ -85,13 +86,15 @@ export function getColorHex(id: string) {
 export const getFbPhoto = (fbId: string) =>
   `https://graph.facebook.com/${fbId}/picture?type=normal`;
 
-export const getMediaType = (input: string) => {
-  if (!input) {
+export const getMediaType = (input: any) => {
```

Comment:
do we have a new non-string input type. . . ?

Reply:
I'll double check, I think it's a leftover when the input could've been a playlist video